independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/UpdateProjectCommandHandler.java

```diff
@@ -70,6 +71,14 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
             logUpdates(currentState, latestCatalog, false, perModule, invocation.getQuarkusProject().log());
 
             if (generateRewriteConfig) {
+                Set<String> additionalSourceFiles = Set.of("**/META-INF/services/**",
```

Comment:
@gsmet  Why not creating the joined string statically?

Reply:
I did that first and found it very unreadable. I don't think it's too much of a problem creating a Set here as we don't really care about one more allocation.